### PR TITLE
4362 system events

### PR DESCRIPTION
**Pull request by johnmcmahon**
- Create two system events on init
- Allow users to get event types by name
- Fix tests to expect two additional eventTypes

The endpoint for EventTypeByName isn't what I wanted it to be but for some reason I couldn't just do /eventType/name/:name because it conflicted with /eventType/:id.  Not sure why soI just made it different to appease the go gods.

### CODE REVIEW

File: workflow/WorkflowServer.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ func (server *WorkflowServer) Init(service *WorkflowService) error {
 
 		{"GET", "/eventType", server.handleGetAllEventTypes},
 		{"GET", "/eventType/:id", server.handleGetEventType},
+		{"GET", "/eventTypeByName/:name", server.handleGetEventTypeByName},
```
Comment (johnmcmahon) [inline workflow/WorkflowServer.go]: Don't really like that but wasn't sure what else to do

File: workflow/WorkflowService.go [hunk 2]
```diff
@@ -142,6 +142,75 @@ func (service *WorkflowService) Init(
...
+		"jobId":    "string",
+		"status":   "string",
+		"dataId":   "string",
+	}
+	executionCompletedType.Mapping = executionCompletedTypeMapping
+	postedExecutionCompletedType := service.PostEventType(executionCompletedType)
+	if postedExecutionCompletedType.StatusCode == 201 {
+		// everything is ok
+	} else {
+		// something is wrong or it was already there
```
Comment (stefanvanburen) [inline workflow/WorkflowService.go:210]: Do we want to have a more explicit failure here? Or at least some logging like the previous check.

**Comment by stefanvanburen** [comment]
Looks good to me other than the note that I made. Fairly subjective though - I know there's other places in workflow that could use an update to how we do logging.

**Comment by johnmcmahon** [comment]
Need to fix some errors, but should be done soon

**Comment by johnmcmahon** [comment]
This should be done now, just need to update the docs to reflect the most recent modifications.

**Pull request merged**